Sources/ShopifyCheckoutSheetKit/CheckoutWebViewController.swift

```diff
@@ -156,6 +155,41 @@ class CheckoutWebViewController: UIViewController, UIAdaptivePresentationControl
 
 		delegate?.checkoutDidCancel()
 	}
+
+	private func presentFallbackViewController(url: URL) {
+		progressObserver?.invalidate()
+		checkoutView.removeFromSuperview()
+
+		self.checkoutView = CheckoutWebView.for(checkout: url, recovery: true)
```

Comment:
Yeah this is somewhat tricky because the CheckoutWebView is instantiated before the `super.init()` call in the `init()` method and you cannot call methods before that instantiation occurs